.gitignore

```diff
@@ -7,6 +7,9 @@
 /bin
 /obj
 
+# vscode ide stuff
+/.vscode
+
```

Comment:
This part is still good.

Reply:
Fixed the merge conflict so it should contain the updated .gitignore. I removed an entry to .vscode because it was duplicated with the merge. Let me know if there are any other changes that need to be made.

Reply:
Looks good! 👍